src/media-server.js

```diff
@@ -51,6 +54,13 @@ export class MediaServer {
       getProject,
     })
 
+    if (staticMapsRootDir) {
+      this.#fastify.register(StaticMapsPlugin, {
+        prefix: MAPS_PREFIX,
+        staticRootDir: staticMapsRootDir,
+      })
```

Comment:
for now we register this plugin directly. can imagine in the future where we want to implement more of the general map server, we'd have a more general "maps" wrapper plugin that would register the static maps plugin, and then register the wrapper here instead.

one thing at a time...